src/main/kotlin/no/nav/familie/ef/søknad/config/FeatureToggleConfig.kt

```diff
@@ -62,6 +62,8 @@ class FeatureToggleConfig(private val enabled: Boolean,
     private fun lagDummyFeatureToggleService(): FeatureToggleService {
         return object : FeatureToggleService {
             override fun isEnabled(toggleId: String, defaultValue: Boolean): Boolean {
+                if (toggleId == "familie.ef.soknad.feilsituasjon")
```

Comment:
Burde ikke denne håneteres i unleash?

Reply:
(DummyToggle) -> Kun hvis unleash er skrudd av

Reply:
Men flytter denne til kun for local env